src/views/Landing.vue

```diff
@@ -0,0 +1,1172 @@
+            {{ $t('views.Landing.section1.linkFeed') }}
+          </RouterLink>
+        </div>
+      </section>
+      <section class="superhero-footer">
+        <div class="sh-container bottom-footer top">
+          <div class="sh-row">
+            <div class="sh-col-6 left-foot">
+              <p>Superhero is the world’s first decentralized, open-source social media
+                platform.</p>
+            </div>
+            <div class="sh-col-6 right-subscribe">
+              <form class="subscribe-form">
+                <div class="form-group">
+                  <input class="form-control" name="" placeholder="Get started" type="text">
+                  <button class="subscribe-button">Subscribe now</button>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+        <div class="sh-container bottom-footer">
+          <div class="sh-row">
+            <div class="sh-col-4 left-sh-foot">
+              <a href=""><img alt="" src="../assets/landing/superhero-logo-black.svg"></a>
+            </div>
+            <div class="sh-col-4 center-sh-foot">
+              <p>© 2020 Superhero.com</p>
+            </div>
+            <div class="sh-col-4 right-sh-foot">
+              <a href="">Privacy</a>
+              <a href="">Terms</a>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+  </div>
+</template>
+<script>
+import VueAos from 'vue-aos';
+
+export default {
+  components: { VueAos },
+  metaInfo: {
+    title: 'Welcome',
+  },
+};
+</script>
+<style lang="scss">
+:root {
+  --animate-delay: 0.1s; /* Animation delay 10x faster */
+  --animate-duration: 0.5s;
+}
+
+.landing-page {
+  background: #f2f2f2;
+  color: #212529;
+  font-family: Oxygen, sans-serif;
+  font-weight: 400;
+  line-height: 1.5;
+  margin: 0 auto;
+  max-width: 100vw !important;
+  text-align: left;
+}
+</style>
+<style lang="scss" scoped>
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+h1 {
+  font-family: IBM Plex Sans, sans-serif;
+  font-size: 56px;
+  font-style: normal;
+  font-weight: 400;
+  margin: 0;
+}
+
+h2 {
+  font-family: IBM Plex Sans, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  margin: 0;
+}
+
+h3 {
+  font-family: Oxygen, sans-serif;
+  font-size: 12px;
+  letter-spacing: 0.8px;
+  line-height: 14px;
+  margin: 0 0 12px;
+  opacity: 0.6;
+  text-shadow: 0 0 0 #000;
+  text-transform: uppercase;
+}
+
+img {
+  max-width: 100%;
+  vertical-align: top;
+}
+
+.sh-container-fluid {
+  margin: 0 auto;
+  max-width: 1660px;
+  padding-left: 15px;
+  padding-right: 15px;
+  width: 100%;
+}
+
+.sh-container {
+  margin: 0 auto;
+  max-width: 1180px;
+  overflow: hidden;
+  padding-left: 15px;
+  padding-right: 15px;
+  width: 100%;
+}
+
+.sh-row {
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  margin-left: -15px;
+  margin-right: -15px;
+}
+
+.sh-col-3 {
+  -ms-flex: 0 0 25%;
```

Comment:
vue cli has autoprefixer enabled by default, so we shouldn't use prefixes by ourselves